ahorcado.rb

```diff
@@ -1,62 +1,44 @@
-#TODO: word_spaces should depend from current_word
-# Array equivalent to the characters of the word 
-# [“”, “”, “”, “”, ...]
-# []
-# Array.new
-
-#current_word = "love"
-#word_spaces = Array.new(current_word.length, "")
-
-@current_word = "friends"
-@word_spaces = Array.new(@current_word.length, "")
-@current_characters = @current_word.split("")
-
-def display_game
-  @word_spaces.each do |characters|
-    print "| #{characters} | "
+class Ahorcado
+  def initialize(current_word)
+    @current_word = current_word
```

Comment:
To make sure, you'll be creating a new "Ahorcado" Object per each game instance, right?

Reply:
Yes, since this is a one-player game, a new game should be created again once the player either wins or loses.